api/api_autogen/library/defaults/Communitysolar_PVWattsCommunitySolar.json

```diff
@@ -34,7 +34,7 @@
 			"reserves_interest": 1.75,
 			"salvage_percentage": 0,
 			"state_tax_rate": [7.000000],
-			"system_capacity": 100000,
+			"system_capacity": 3000,
```

Comment:
Maybe I missed something, but I think community solar costs might need some attention. They don't seem consistent with either commercial or utility, but also are not reliably in between (by individual cost category)

Reply:
Fixed in latest push